### PR TITLE
FAI-10152 - Sprint history record in sprint_reports stream

### PR DESCRIPTION
**Pull request by matiaslcoulougian**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/jira-source/src/jira.ts [hunk 1]
```diff
@@ -772,7 +782,8 @@ export class Jira {
 
   async getSprintReport(
     sprint: AgileModels.Sprint,
-    boardId: string
+    boardId: string,
+    includeIssues = true
```
Comment (chalenge) [inline sources/jira-source/src/jira.ts:786]: when do we ever not want to include the report issues?

File: destinations/airbyte-faros-destination/test/converters/jira.test.ts [hunk 2]
```diff
@@ -96,7 +96,8 @@ describe('jira', () => {
       tms_Project: 1,
       tms_ProjectReleaseRelationship: 3,
       tms_Release: 3,
-      tms_Sprint: 13,
+      tms_Sprint: 14,
+      tms_SprintHistory: 6,
```
Comment (chalenge) [inline destinations/airbyte-faros-destination/test/converters/jira.test.ts:100]: in a separate PR let's start having real data objects written see [Asana tests](https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/test/converters/asana.test.ts#L122-L126)

**Review by chalenge — APPROVED** [review]
See comments

File: sources/jira-source/src/jira.ts (hunk 1 above)
Comment (matiaslcoulougian) [reply sources/jira-source/src/jira.ts:786]: For WebhookSupplement mode we only need the sprint report point fields, to write this fields in `sprint` record. `sprint_history` record will already be [populated](https://github.com/faros-ai/hermes/blob/20d3d52564e5bfc9ee71fbf59788b836503791b6/packages/worker/src/converters/jiraConverter.ts#L370) with webhook issue events.

File: destinations/airbyte-faros-destination/test/converters/jira.test.ts (hunk 2 above)
Comment (matiaslcoulougian) [reply destinations/airbyte-faros-destination/test/converters/jira.test.ts:100]: https://faros-ai.atlassian.net/browse/FAI-10982

File: sources/jira-source/src/jira.ts (hunk 1 above)
Comment (chalenge) [reply sources/jira-source/src/jira.ts:786]: No, the [fields populated from the report](https://github.com/faros-ai/airbyte-connectors/pull/1426/files#diff-087c79ba42f4c18f101492fab2931ed8b6d7128048719e0479503a20598428a9R38-R40) are different from those that we get [from the issue changelog](https://github.com/faros-ai/hermes/blob/20d3d52564e5bfc9ee71fbf59788b836503791b6/packages/worker/src/converters/jiraConverter.ts#L429-L430)

**Review by chalenge — CHANGES_REQUESTED** [review]
We should sync the issue sprint report data

File: sources/jira-source/src/jira.ts (hunk 1 above)
Comment (matiaslcoulougian) [reply sources/jira-source/src/jira.ts:786]: Missed that. Moving to fast 🙂

**Review by chalenge — APPROVED** [review]

**Pull request merged**